vision/manipulation/super_resolution/README.md

```diff
@@ -0,0 +1,58 @@
+# SuperResolution
+
+## Use cases
+The Super Resolution machine learning model sharpens and upscales the input image to refine the details and improve quality.
+
+## Description
+Super Resolution uses efficient  [Sub-pixel convolutional layer](https://arxiv.org/abs/1609.05158) described for increasing spatial resolution within network tasks. By increasing pixel count, images are then clarified, sharpened, and upscaled without losing the input image’s content and characteristics. 
+
+## Model
+
+ |Model      |Download  |Checksum|Download (with sample test data)| ONNX version |
+|-------------|:--------------|:--------------|:--------------|:--------------|
+|Super_Resolution v1|    [240 KB](super_resolution_tutorial.onnx)  |[MD5](super_resolution-md5.txt)  |  [7.6 MB](super_resolution_test_image.gz) |  1.5.0  |
+
+## Inference
+
+
+### Input 
+Image file can be jpg, png, and jpeg and its input sizes are dynamic. The inference was done using jpg image.
+
+### Preprocessing
+Images are resized into (224x224). The image is then split into ‘YCbCr’ color components: greyscale ‘Y’, blue-difference  ‘Cb’, and red-difference ‘Cr’. Once the greyscale Y component is extracted, it is then converted to tensor and used as the input image.
```

Comment:
Why is the size restricted to 224x224? Is it because of the pytorch tracing export?

Reply:
Yea, its due to the tracing export.